pkg/registry/core/persistentvolume/strategy_test.go

```diff
@@ -34,3 +41,398 @@ func TestSelectableFieldLabelConversions(t *testing.T) {
+		},
+		{
+			name: "feature disabled: user timestamp change is respected on phase change",
+			fg:   false,
+			oldObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumePending,
+					LastPhaseTransitionTime: &origin,
+				},
+			},
+			newObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumeBound,
+					LastPhaseTransitionTime: &later,
+				},
+			},
+			expectedObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumeBound,
+					LastPhaseTransitionTime: &later,
+				},
+			},
+		},
+		{
+			name: "feature disabled: user timestamp change is respected on no phase change",
+			fg:   false,
+			oldObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumeBound,
+					LastPhaseTransitionTime: &origin,
+				},
+			},
+			newObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumeBound,
+					LastPhaseTransitionTime: &later,
+				},
+			},
+			expectedObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumeBound,
+					LastPhaseTransitionTime: &later,
+				},
+			},
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PersistentVolumeLastPhaseTransitionTime, tc.fg)()
+
+			obj := tc.newObj.DeepCopy()
+			StatusStrategy.PrepareForUpdate(context.TODO(), obj, tc.oldObj.DeepCopy())
+			if !reflect.DeepEqual(obj, tc.expectedObj) {
+				t.Errorf("object diff: %s", cmp.Diff(obj, tc.expectedObj))
+			}
+		})
+	}
+}
+
+func TestStatusCreate(t *testing.T) {
+	now := metav1.Now()
+	nowFunc = func() metav1.Time { return now }
+	defer func() {
+		nowFunc = metav1.Now
+	}()
+	tests := []struct {
+		name        string
+		fg          bool
+		newObj      *api.PersistentVolume
+		expectedObj *api.PersistentVolume
+	}{
+		{
+			name: "feature enabled: pv is in pending phase and has a timestamp",
+			fg:   true,
+			newObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+			},
+			expectedObj: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				Status: api.PersistentVolumeStatus{
+					Phase:                   api.VolumePending,
+					LastPhaseTransitionTime: &now,
+				},
+			},
+		},
+		{
+			name: "feature disabled: pv does not have phase and timestamp",
+			fg:   true,
```

Comment:
This should be flipped to false, and the status should be empty

Reply:
Corrected.